CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ And please only add new entries to the top of this list, right below the `# Unre
 - On Wayland, fix rare crash on DPI change
 - Web: Added support for `Window::theme`.
 - On Wayland, fix rounding issues when doing resize.
+- On macOS, resize simple fullscreen windows on windowDidChangeScreen events
```

Comment:
Could you revert arbitrary changes you've done to this file besides this line? Thx.

Reply:
sorry didn't notice my editor did that